src/components/audio-feedback.js

```diff
@@ -169,7 +169,7 @@ AFRAME.registerComponent("scale-audio-feedback", {
 
   async init() {
     await waitForDOMContentLoaded();
-    this.camera = document.getElementById("viewing-camera").object3D;
```

Comment:
What was the motivation / reason for changing all these spots to use the camera object in the `object3DMap`?

Reply:
Oops, nevermind I see you explained it in the summary!